styles/style.css

```diff
@@ -723,15 +706,374 @@
   footer hr {
     width: 30%;
     margin-left: 35%;
+    margin-top: 20px;
     height: 6px;
     border-radius: 8px;
     background: black;
   }
 }
 
+/* ========================================================================================================================= */
+
+/* ===============================    Desktop Version Portfolio ============================================================ */
+
 /* Desktop scream size minimum is 768 px */
 @media (min-width: 768px) {
   body {
     /* Nothing to display for the moment */
   }
+
+  .toolbar {
+    /* background: red; */
```

Comment:
- [ ] kindly consider removing the commented pice of codes cause they make your code base look messy

Reply:
Ok done